README.md

```diff
@@ -1,45 +1,80 @@
+A simplified tabletop role-playing game with an emphasis on casual play.
 
-Cardmaster is a simplified tabletop roleplaying card game which emphasizes casual play.
+## The Card System
 
-Players play as "Cardmasters", who have a hand of single-use cards which allow them to use magical abilities at **any** time, even if it isn't their turn.
+As a *Cardmaster*, the player has a hand of single-use cards which allow them to use magical abilities instantly *at any time* during play.
+<p align="center">
+  <img src="https://git.io/JEQYa" width="30%" title="Boost">
+</p>
 
-Consider the card "Shift":
+Since the **⚡ Cast Effect** applies instantly, the seemingly lackluster _Shift_ could be used to dodge a bullet, if played on an opponent's turn. 
 
-<img src="https://user-images.githubusercontent.com/8680290/127941236-751beb93-ac1f-4605-91eb-8e6364d08cfb.png"  width="33%" title="⚡ Move a metre">
+In the same situation, cards such as *Fluster*, *Gust of Wind*, and *Anvil Drop* could be used to interfere.
 
-At first glance, this ability is underwhelming. However, when cast just as an enemy makes an attack, it acts as a reliable escape. The effect of a card applies instantaneously, so "Shift" can even be used to dodge bullets.
+<br>
+<p align="center">
+  <img src="https://raw.githubusercontent.com/elliottomlinson/cardmaster/master/res/card/generated/Fluster.png" width="25%" title="Fluster">
+  <img src="https://raw.githubusercontent.com/elliottomlinson/cardmaster/master/res/card/generated/Gust%20of%20Wind.png" width="25%" title="Gust of Wind">
+    <img src="https://raw.githubusercontent.com/elliottomlinson/cardmaster/master/res/card/generated/Anvil%20Drop.png" width="25%" title="Anvil Drop">
+</p>
 
-### Cards Have Many Uses:
+During an ally's turn, *Field Medicine* and *Pigeon Partner* could be cast to help them. Cards like *Lie* and *Seance* can affect social interactions and may be most useful outside of combat.
 
-During an opponent's turn, players can look for ways to interrupt their plans with cards like "Distract":
+<br>
+<p align="center">
+  <img src="https://raw.githubusercontent.com/elliottomlinson/cardmaster/master/res/card/generated/Field%20Medicine.png" width="20%" title="Field Medicine">
+  <img src="https://raw.githubusercontent.com/elliottomlinson/cardmaster/master/res/card/generated/Pigeon%20Partner.png" width="20%" title="Pigeon Partner">
+  <img src="https://raw.githubusercontent.com/elliottomlinson/cardmaster/master/res/card/generated/Lie.png" width="20%" title="Lie">
+  <img src="https://raw.githubusercontent.com/elliottomlinson/cardmaster/master/res/card/generated/Seance.png" width="20%" title="Seance">
+</p>
 
-<img src="https://user-images.githubusercontent.com/8670351/131440331-e310e151-9064-4413-b135-86574c2fafd0.png"  width="33%">
+Enabled by our mapping system, players have the freedom to manpulate the environment to varying degrees with cards like *Eidolimb*, *Soak*, and *Poltergeist*.
```

Comment:
Should the mapping system be a part of Cardmaster?

Reply:
I'm up for editing it out of this line just to cover all of our bases